### PR TITLE
Add app version into settings

### PR DESCRIPTION
**Pull request by joshxg**
image

🤷 !

### CODE REVIEW

**Comment by mark-prins** [comment]
but that looks so professional! 😄 
can we have it when you tap it twelve times, the app starts scrolling the list of developers?

File: src/ui/hooks/useDependency.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useContext } from 'react';
 import { Dependency, DependencyKey } from '~services/DependencyLocator/DependencyLocator';
-import { DependencyLocatorContext, FormatService, MigrationService } from '~services';
+import { DependencyLocatorContext, FormatService, MigrationService, UtilService } from '~services';
 import { DEPENDENCY } from '~constants';
```
Comment (mark-prins) [inline src/ui/hooks/useDependency.ts:4]: This can be removed now

File: src/ui/hooks/useDependency.ts [hunk 2]
```diff
@@ -11,11 +11,16 @@ export const useDependency = (service: DependencyKey): Dependency | undefined =>
 };
 
 export const useMigrationService = (): MigrationService => {
-  const migrationService = useDependency(DEPENDENCY.MIGRATION);
+  const migrationService = useDependency('migrationService');
```
Comment (mark-prins) [inline src/ui/hooks/useDependency.ts:14]: the `DEPENDENCY.MIGRATION` can be removed from consts

File: src/common/services/UtilService/UtilService.ts [hunk 3]
```diff
@@ -1,11 +1,14 @@
 import moment from 'moment';
 import generateUUID from 'react-native-uuid';
+import packageJson from '../../../../package.json';
```
Comment (mark-prins) [inline src/common/services/UtilService/UtilService.ts:3]: maybe..
```suggestion
import packageJson from '~/../../package.json';
```

**Review by mark-prins — APPROVED** [review]
👍

**Comment by joshxg** [comment]
Thanks- i've done the suggestions in #167

**Pull request merged**